src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaFailureDetails.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import {t} from 'app/locale';
+
+type Props = {
+  lambdaFunctionFailures: Array<{FunctionName: string; Runtime: string}>;
+};
+
+export default function AwsLambdaFailureDetails({lambdaFunctionFailures}: Props) {
+  return (
+    <Wrapper>
+      <h3>{t('Failed to update the following Lambda Functions')}</h3>
+      {lambdaFunctionFailures
+        .sort((a, b) =>
```

Comment:
@scefali Since we are not using this endpoint for anything else, I think we should sort `lambdaFunctionFailures` on the server and return the result.